config/static-assets/static/regulations/css/scss/module/_custom.scss

```
@@ -1 +1,45 @@
 /* Placeholder for agency-specific modules (i.e. features) */
+
+.recent-changes {
+    margin-top: 30px;
+    width: 100%;
+    height: 175px;
```

On small screens the text is overflowing the box, it looks like changing this to `min-height` fixes things
```suggestion
    min-height: 175px;
```